CHANGELOG.md

```diff
@@ -1,3 +1,172 @@
+# 1.9.0 (September 2025)
+
+_Changes since 1.8.2_
+
+## Highlights
+
+### Multiple Platforms
+
+- `material3` library now includes new experimental `MaterialExpressiveTheme` [#2127](https://github.com/JetBrains/compose-multiplatform-core/pull/2127)
```

Comment:
Not valid anymore (removed by Google in beta). If we need to keep it, it must be combined with "use alpha for this feature" note